### ISSUE TITLE
Multiple different success responses

### ISSUE THREAD

**Issue by marcferna** [ruby-grape/grape-swagger#674]
Is there a way that I can set the `success` responses to be an array like the `failure`? The use case that we have is a signup POST endpoint that returns 201 if the user is created and a 200 if the user is already created (logging the user in directly)

Something like:
```
desc 'Create or log in a user',
    success: [
      { code: 201, message: 'user is created' },
      { code: 200, message: 'user is found' }
    ]
```

**Comment by davidsiaw** [comment]
Feels like this issue can be closed since the relevant PR that fixes it has already been merged

**Comment by LeFnord** [comment]
closed via #747
